cachito/workers/tasks/celery.py

```diff
@@ -2,15 +2,45 @@
 import sys
 
 import celery
-from celery.signals import celeryd_init, task_postrun, task_prerun
+from celery.signals import celeryd_init, task_postrun, task_prerun, worker_process_init
+from opentelemetry import trace
+from opentelemetry.exporter.jaeger.thrift import JaegerExporter
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
+from opentelemetry.instrumentation.celery import CeleryInstrumentor
+from opentelemetry.sdk.resources import SERVICE_NAME, Resource
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
 
 from cachito.workers.celery_logging import (
     cleanup_task_logging,
     cleanup_task_logging_customization,
     setup_task_logging,
     setup_task_logging_customization,
 )
-from cachito.workers.config import app, validate_celery_config  # noqa: F401
+from cachito.workers.config import app, get_worker_config, validate_celery_config  # noqa: F401
+
+
+def init_celery_tracing(*args, **kwargs):
```

Comment:
Nitpick: since this will only be used in the same module, we could make it private.